test/integration/relay/connection.test.js

```diff
@@ -479,6 +482,84 @@ describe('relay', function () {
 
     });
 
+    it('should handle orderBy function case', async function () {
+      await graphql(this.schema, `
+        {
+          user(id: ${this.userA.id}) {
+            projects(first: 1) {
+              edges {
+                node {
+                  tasks(orderBy: NAME_FUNC, first: 5) {
+                    edges {
+                      cursor
+                      node {
+                        id
+                        name
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      `, null, {});
+
+      expect(this.projectOrderSpy).to.have.been.calledOnce;
+      expect(this.projectOrderSpy.alwaysCalledWithMatch({}, { first: 5 })).to.be.ok;
+    });
+
+    it('should properly reverse NULLS LAST in orderBy', async function () {
```

Comment:
It still test ASC was reversed to DESC, even in multi order case, should keep it